src/ucp/core/ucp_request.inl

```diff
@@ -176,12 +176,24 @@ UCS_PTR_MAP_IMPL(request, 0);
     }
 
 
+#define UCP_REQUEST_CHECK_OP_ATTR_FLAGS(_param) \
+    if (ENABLE_PARAMS_CHECK && \
+        ucs_test_all_flags((_param)->op_attr_mask, \
+                           (UCP_OP_ATTR_FLAG_FAST_CMPL | \
+                            UCP_OP_ATTR_FLAG_MULTI_SEND))) { \
+        ucs_error("invalid optimization flags combination: %u", \
+                  (_param)->op_attr_mask); \
```

Comment:
Why make the user guess?

`ucs_error("UCP_OP_ATTR_FLAG_FAST_CMPL and UCP_OP_ATTR_FLAG_MULTI_SEND are mutually exclusive");`